secure_message/repository/database.py

```diff
@@ -157,6 +170,37 @@ def serialize(self):
         return data
 
 
+class Actors(db.Model):
+    """Label Assignment table model"""
+    __tablename__ = "actors"
+
+    id = Column("id", Integer(), primary_key=True)
+    msg_id = Column('msg_id', String(constants.MAX_MSG_ID_LEN + 1), ForeignKey('secure_message.msg_id'), index=True)
+    from_actor = Column('from_actor', String(constants.MAX_STATUS_ACTOR_LEN + 1))
+    to_actor = Column('to_actor', String(constants.MAX_STATUS_ACTOR_LEN + 1))
+    sent_from_internal = Column('sent_from_internal', Boolean())
+
+    def __init__(self, msg_id, from_actor, to_actor, sent_from_internal):
+        self.set_from_domain_model(msg_id, from_actor, to_actor, sent_from_internal)
+
+    def set_from_domain_model(self, msg_id, from_actor, to_actor, sent_from_internal):
+        """Set actors values"""
+        self.msg_id = msg_id
+        self.from_actor = from_actor
+        self.to_actor = to_actor
+        self.sent_from_internal = sent_from_internal
+
+    @property
+    def serialize(self):
```

Comment:
Does serialize actually serialize, or just return data suitable for serialization?

The name suggests the former; the comment suggests the latter

Reply:
They return the data in a serialisable format . This pattern is used on all domain classes . AId rather leave it in until the team agrees on this one

Reply:
Probably should add an issue or something to deal with clarifying that going forward. Fair enough for consistency in the short term.